### PR TITLE
Momentary Assessments

### PR DESCRIPTION
**Pull request by golf1052**
Implemented logic for prompting users to do momentary assessments.
Addresses #8

### CODE REVIEW

**Comment by ColinKohler** [comment]
Can you add java doc comments on this? Just so all our code looks the same. Also how does testing this work?

**Comment by golf1052** [comment]
Added code to address #15 
Added JavaDocs.
Added a test for calendar creation (related to creating alarms).

File: app/src/main/java/justbe/mindfulnessapp/AlarmReceiver.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+public class AlarmReceiver extends BroadcastReceiver {
+    /**
+     * Performs an action after receiving an alarm
+     * @param context The app context
+     * @param intent The intent
+     */
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        String action = intent.getAction();
+        if (action != null && action.equals("android.intent.action.BOOT_COMPLETED")) {
```
Comment (ColinKohler) [inline app/src/main/java/justbe/mindfulnessapp/AlarmReceiver.java:22]: Why, dear god, would you do this... Just use the negation of this statement...

**Comment by hklebs** [comment]
You should add more comments within the functions to explain what you are doing and you can revise  the if/else in onReceive, you check action != null twice and you could revise the first if/else to not have the else case

File: app/src/main/java/justbe/mindfulnessapp/AlarmReceiver.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                                .setContentText("Pending morning assessment");
+                        // TODO: go to the correct morning assessment
+                        PendingIntent contentIntent = PendingIntent.getActivity(context, 0,
+                                new Intent(context, MainActivity.class), 0);
+                        notificationBuilder.setContentIntent(contentIntent);
+                        NotificationManager notificationManager = (NotificationManager)context.getSystemService(Context.NOTIFICATION_SERVICE);
+                        notificationManager.notify(001, notificationBuilder.build());
+                    }
+                    else {
+                        NotificationCompat.Builder notificationBuilder = new NotificationCompat.Builder(context)
```
Comment (ColinKohler) [inline app/src/main/java/justbe/mindfulnessapp/AlarmReceiver.java:44]: Daisy chaining stuff like this is really really bad also...

**Pull request merged**